src/contexts/EnvironmentContext.js

```diff
@@ -38,6 +38,12 @@ export const EnvironmentProvider = ({ children }) => {
             usersTitle: "Users",
             zonesTitle: "Zones",
 
+            groupsPerPageKey: "zmt-perpage-groups",
+            resourcesPageKey: "zmt-perpage-resources",
+            serversPageKey: "zmt-perpage-servers",
+            usersPageKey: "zmt-perpage-users",
+            defaultAmount: 25,
```

Comment:
need a more specific name here... perhaps... `defaultPageSize` or `defaultItemsPerPage`

Reply:
i really like `defaultItemsPerPage`. changed to that

Reply:
bueno